stubs/pika/@tests/stubtest_allowlist.txt

```diff
@@ -3,9 +3,8 @@ pika.compat.StringIO.seek
 pika.compat.StringIO.truncate
 
 # Requires external libraries to be installed.
-pika\.adapters\.gevent_connection.*
-pika\.adapters\.tornado_connection.*
-pika\.adapters\.twisted_connection.*
+pika.adapters.gevent_connection
```

Comment:
Any reason not to add `gevent` to `requirements-stubtest.txt` too, and get some stubtest coverage for `gevent_connection.pyi`?

Reply:
afaik gevent is not type. But maybe that doesn't matter for stubtest?
I know we don't use untyped `requires`. But if it's fine for stubtest so it can scan a file then I'll add it.

Reply:
Stubtest attempts to dynamically import every module it can find at runtime. But it currently can't import `pika.adapters.gevent_connection`, because `gevent_connection.py` has `import gevent` at the top of the file. The requirements in `requirements-stubtest.txt` don't need to be typed; they're just there to help stubtest out in situations like this, where doing `pip install pika` isn't sufficient for stubtest to be able to do its dynamic runtime inspection.

(Arguably `pika` should probably specify a `gevent` "extra" at runtime, so that we could tell stubtest to do `pip install pika[gevent]`, rather than hacking about with extra `requirements-stubtes.txt` files. But it is what it is.)